mlrun/projects/project.py

```diff
@@ -2008,10 +2008,10 @@ def _instantiate_model_monitoring_function(
 
     def enable_model_monitoring(
         self,
-        default_controller_image: str = "mlrun/mlrun",
         base_period: int = 10,
         image: str = "mlrun/mlrun",
         deploy_histogram_data_drift_app: bool = True,
+        default_controller_image: str = "mlrun/mlrun",
```

Comment:
Please revert this - the order of the params have to stay as it was for BC

Reply:
The order was not changed :)